lib/shopify-cli/messages/messages.rb

```diff
@@ -172,6 +172,7 @@ module Messages
             {{x}} error: Your account was not found. Please sign up at https://partners.shopify.com/signup
             For authentication issues, run {{command:%s logout}} to clear invalid credentials
             MESSAGE
+            internal_server_error: '{{red:{{x}} An unexpected error occurred on Shopify.}}',
```

Comment:
@amanbis @iainmcampbell  is this a good internal server error message? we made it up

Reply:
I think this is ok as a generic fallback for unhandled exceptions.